src/software/jetson_nano/ansible/playbooks/remote_flash.yml

```diff
@@ -0,0 +1,77 @@
+---
+
+- name: Remote flashing
+  hosts: THUNDERBOTS_HOSTS
+  vars:
+   all_services:
+     - thunderloop
+     - display
+     - announcement
+     - redis
+   all_actions:
+     - stop
+     - sync
+     - start
+   binary_paths:
+     thunderloop: ../../../../../../../../jetson_nano/thunderloop_main
+     announcement: ../../../../../../../../jetson_nano/broadcasts
+     redis: ../../../../../../../../jetson_nano/redis
+     display: ../../../../../../../../jetson_nano/display
```

Comment:
you only need to go back twice (i.e `../../`) and you can remove the jetson_nano part

Reply:
changed